ci/e2ereport.sh

```diff
@@ -9,7 +9,9 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
 
 if [ -z "${NO_OPEN:-}" ]; then
   if [ -s "$REPORT_OUTPUT" ]; then
-    open "$REPORT_OUTPUT"
+    open "$REPORT_OUTPUT" 2>/dev/null || \
```

Comment:
Use `command -v` to check if they exist before opening them.

With `2>/dev/null` you won't get any error messages even if they're legit.

Reply:
Grep other scripts for `command -v` to see how it's used.

Reply:
Addressed on e8906ef :D